addons/snailmail/static/tests/chatter_snailmail_tests.js

```diff
@@ -0,0 +1,499 @@
+odoo.define('snailmail.chatter_snailmail_tests', function (require) {
+"use strict";
+
+var FormView = require('web.FormView');
```

Comment:
`mail` then `web` then property of module:
```js
var mailTestUtils = require('mail.testUtils');
var ThreadWidget = require('mail.widget.Thread');

var FormView = require('web.FormView');
var testUtils = require('web.test_utils');

var createView = testUtils.createView;
```
(empty line between group eases readability)